src/main/java/me/zeroeightsix/kami/module/modules/player/Freecam.kt

```diff
@@ -36,6 +36,11 @@ object Freecam : Module() {
     private val autoRotate = register(Settings.b("AutoRotate", true))
     private val arrowKeyMove = register(Settings.b("ArrowKeyMove", true))
     private val disableOnDisconnect = register(Settings.b("DisconnectDisable", true))
+    private val directionMode = register(Settings.e<DirectionMode>("DirectionMode", DirectionMode.SpaceUp))
+
+    private enum class DirectionMode {
+        SpaceUp, Look
```

Comment:
The naming for enum should be all UPPER CASE. And it would be better to name them as CREATIVE and 3D. For now just call it THREE_DEE because you can't name something begin with a number.

Reply:
How about `SOURCE` (in reference to Source games) and `CREATIVE`?

Reply:
someone make a decision. i associate superman with this for some reason

Reply:
i described the idea as source engine noclip (so Source) but i just stole the idea from Seppuku anyway so idk

Reply:
I mean I know *of* the idea from source games, source makes more sense

Reply:
> How about `SOURCE` (in reference to Source games) and `CREATIVE`?

That sounds more weird and make less sense

Reply:
> I mean I know _of_ the idea from source games, source makes more sense

Not really, it is been a thing since 20 years ago and labeling flight control with the name of a game engine is just ridiculous

Reply:
And I am pretty sure that 90% of Minecraft player have no idea wtf "SOURCE" is and even player playing Source engine game don't really know what the name of the game engine or "SOURCE" flight is. Other clients just call it 3D and it surely makes more sense than a name of a game engine